src/api/print.api.js

```diff
@@ -39,6 +39,20 @@ class GeoAdminCustomizer extends BaseCustomizer {
         // Call parent layerFilter method for other layers
         return super.layerFilter(layerState)
     }
+
+    /**
+     * Remove the "editableFeature" adn "geodesic" property from the feature as it is not needed and
+     * can cause issues with mapfishprint
+     *
+     * @param {State} layerState
+     * @param {GeoJSONFeature} feature Manipulated feature
+     */
+    feature(layerState, feature) {
+        // cause circular reference issues
+        delete feature.properties.geodesic
+        // unnecessary properties for printing and cause mapfishprint to throw an error
+        delete feature.properties.editableFeature
```

Comment:
This could lead to a type error if `feature.properties` does not exists, maybe use `delete feature.properties?.geodesic`